src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2Engine.java

```diff
@@ -578,28 +407,22 @@ protected int getCountOfNonRefEvents(final ReadPileup pileup, final byte refBase
     }
 
     private double calculateF(final ReadPileup pileup, final byte refBase, final byte minBaseQual) {
```

Comment:
I think it's slightly better to use streams here. Taking your code from elsewhere in the project:

```java
int totalCount = (int) StreamSupport.stream(pileup.spliterator(), false).count();
int altCount = (int) StreamSupport.stream(pileup.spliterator(), false)
        .filter(p -> p.isDeletion() || p.getQual() > minBaseQual)
        .filter(p -> isNonRef(refBase, p))
        .count();
return (double) altCount / totalCount;
```